src/main/java/calculator/Calculator.java

```diff
@@ -0,0 +1,145 @@
+package calculator;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Stack;
+import java.util.StringTokenizer;
+
+class Node {
+    String value;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/app/thejerome/IFMO_Servlets_20190915/pullRequest?prid=4510206)